### DISCUSSION TITLE
Customize form from bot redirect

### DISCUSSION THREAD

**Discussion by PilarRoBr** [LiveHelperChat/livehelperchat#1913] [Q&A]
Hi! I recently began installing Live Helper Chat. I believe I have everything ready to start, but I'm having an issue: I modified the inicial form to add two fields we need. That's working fine.
However, when someone requests a transfer from the chatbot and there are no online operators, the form shown has only the original fields, and doesn't show the extra ones.
This is the trigger to leave a message:

![imagen](https://user-images.githubusercontent.com/125928165/220204419-46fd42eb-43c0-40ed-a728-69c99db0d606.png)

I don't see a way to add custom fields to the offline form.

Thanks for the help!

**Comment by remdex** [comment]
https://doc.livehelperchat.com/docs/chat/start-chat-form-settings
